repo/packages/F/flink/1/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "packagingVersion": "3.0",
+
+    "name": "flink",
+    "version": "1.2.0-1.0",
+    "description": "Apache Flink® is an open source platform for distributed stream and batch data processing.",
+    "framework": true,
+    "website": "http://flink.apache.org/",
+    "tags": [
+        "stream-processing",
+        "batch"
+    ],
+    "scm": "https://github.com/apache/flink",
+    "licenses": [
+        {
+            "name": "Apache License Version 2.0",
+            "url": "https://raw.githubusercontent.com/apache/flink/master/LICENSE"
+        }
+    ],
+
+    "maintainer": "support@mesosphere.io",
```

Comment:
it should be data artisans support email if they are supporting it. If not, change it to dcos.io/community/